api/v1alpha2/virtualmachine_types.go

```diff
@@ -338,15 +338,19 @@ type VirtualMachineSpec struct {
 	// +optional
 	Reserved *VirtualMachineReservedSpec `json:"reserved,omitempty"`
 
-	// MinHardwareVersion specifies the desired minimum hardware version
-	// for this VM.
+	// +optional
```

Comment:
Did you intend to move these above the description comment? Typically we have them after it.

Reply:
It was intentional. I was going to slowly move the rest over time. That way generated GoDocs don't include the kubebuilder annotations. If you feel strongly, we can leave this alone and move them all at once later.

Reply:
OK. This is fine with me.